sdk/terraform/Azure.ResourceManager.Terraform/tests/Tests/ExportTerraformTests.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Terraform.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Terraform.Tests
+{
+    public class ExportTerraformTests : TerraformManagementTestBase
+    {
+        private ResourceGroupResource _resourceGroup;
+
+        public ExportTerraformTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        [SetUp]
+        public async Task SetUp()
+        {
+            _resourceGroup = await CreateResourceGroup(DefaultSubscription, "exportTerraformRg", AzureLocation.WestUS);
+        }
+
+        [TearDown]
+        public async Task TearDown()
+        {
+            await _resourceGroup.DeleteAsync(WaitUntil.Completed);
+        }
+
+        [TestCase]
+        [RecordedTest]
+        public async Task ExportTerraformTest()
+        {
+            string rgName = _resourceGroup.Data.Name;
+            ArmOperation<TerraformOperationStatus> operationStatus = await DefaultSubscription.ExportTerraformAsync(WaitUntil.Completed, new ExportResourceGroup(rgName));
+            string hcl = operationStatus.Value.Properties.Configuration;
+
+            Assert.That(hcl, Does.Contain("azurerm_resource_group"));
+            Assert.That(hcl, Does.Contain(rgName));
```

Comment:
would it be better that we just assert it has `azurerm_resource_group {rgName}` in the result?
We should be expecting they appear in the same line, right?

Reply:
That will make the test more brittle. The actual string returned to us is full of escape characters and whitespaces that might change in future versions (see example: [raw](https://gist.github.com/gerrytan/87788fbe9eed712f84f1d4ef22d3917c#file-raw-txt) / [formatted](https://gist.github.com/gerrytan/87788fbe9eed712f84f1d4ef22d3917c#file-formatted-tf)). The resource group name is a child element.